File: C/practicals/Practical-3.c/p9.c
#include<stdio.h>

void main(){
	 int a,b,c,x,y;
	 printf("Enter the values of a,b and c: \n");
	 scanf("%d %d %d",&a,&b,&c);

	 if(a*a+b*b==c*c){
		 printf("\n it is PYTHAGOREAN TRPLATE\n");
	 }else {
		 printf("\n it is NOT PYTHAGOREAN TRPLATE\n");
	 }
}

File: JAVA/AsgnPract/4/p9.java
import java.util.*;

class nine{
	
	public static void main(String[] args){


		Scanner py = new Scanner(System.in);

		System.out.print("Enter the value of a: ");
		int a = py.nextInt();
		
		System.out.print("Enter the value of b: ");
		int b = py.nextInt();
		
		System.out.print("Enter the value of c: ");
		int c = py.nextInt();

		if (a*a + b*b == c*c){

			System.out.println("Its Pythagorean Triplate");
		}else{
			System.out.println("Not a Pythagorean Triplate");
		}
		py.close();
	}
}



